src/pyodide/python-entrypoint-helper.ts

```diff
@@ -255,18 +255,28 @@ function makeEntrypointProxyHandler(
       const isKnownHandler = SPECIAL_HANDLER_NAMES.includes(prop);
       const isKnownDoHandler =
         isDurableObject && SPECIAL_DO_HANDLER_NAMES.includes(prop);
+      const isFetch = prop == 'fetch';
       if (isKnownHandler || isKnownDoHandler) {
         prop = 'on_' + prop;
       }
 
       return async function (...args: any[]): Promise<any> {
         // Check if the requested method exists and if so, call it.
         const pyInstance = await pyInstancePromise;
-        if (typeof pyInstance[prop] === 'function') {
-          return await doPyCallHelper(isKnownHandler, pyInstance[prop], args);
-        } else {
+        if (typeof pyInstance[prop] !== 'function') {
           throw new TypeError(`Method ${prop} does not exist`);
         }
+
+        if ((isKnownHandler || isKnownDoHandler) && !isFetch) {
+          return await doPyCallHelper(true, pyInstance[prop], args);
+        }
```

Comment:
Is this right? Don't we want to apply some wrapper logic to all RPC calls?

Reply:
For `alert` at least we don't. I guess we could, but the object that's passed in just has two fields that can be easily read using Pyodide's FFI.

For the websocket methods we would need to implement wrappers for them, so keeping them unwrapped for now makes sense. But even for those I think the arguments are mostly usable via the FFI: https://developers.cloudflare.com/durable-objects/api/base/#websocketmessage. The WebSocket object is something we may want to wrap, but that will require further work.